File: Plugins/Aspose_Pdf_Java_for_PHP/src/Aspose/Pdf/WorkingWithDocumentConversion/PdfToDoc.php
<?php

namespace Aspose\Pdf\WorkingWithDocumentConversion;

use com\aspose\pdf\Document as Document;

class PdfToDoc {

    public static function run($dataDir=null)
    {
        # Open the target document
        $pdf = new Document($dataDir . 'input1.pdf');

        # Save the concatenated output file (the target document)
        $pdf->save($dataDir . "output.doc");

        print "Document has been converted successfully" . PHP_EOL;
    }

}

File: Plugins/Aspose-Pdf-Java-for-Jython/asposepdf/WorkingWithDocumentConversion/PdfToDoc.py
from asposepdf import Settings
from com.aspose.pdf import Document

class PdfToDoc:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithDocumentConversion/PdfToDoc/'
        
        # Open the target document
        pdf = Document(dataDir + 'input1.pdf')

        # Save the concatenated output file (the target document)
        pdf.save(dataDir + "output.doc")

        print "Document has been converted successfully"
 
if __name__ == '__main__':        
    PdfToDoc()